server/controllers/login.js

```diff
@@ -13,19 +13,19 @@ exports.login = (req, res) => {
   })
```

Comment:
this file should not be uploaded with this branch :/

Reply:
haha, so delete ```prettier``` module tho